cli/EAP7-984_Search_Output.adoc

```diff
@@ -0,0 +1,70 @@
+= EAP7-984 Search output for CLI
+:author:            JF Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse/
+
+== Overview
+
+The intent of this feature is to provide a search capability to the CLI. When output is longer 
+than terminal height, CLI pages the display and offers a workflow similar to "more" to navigate the content. 
+The proposed feature extends this workflow with the ability to search for text inside the displayed content.
+
+== Issue Metadata
+
+=== Issue:
+
+* {issue-base-url}EAP7-984[EAP7-984]
+
+=== Related Issues:
+
+* {issue-base-url}WFCORE-3743
+* {issue-base-url}WFCORE-3568 (dependency on this one).
+
+=== Dev Contacts:
+
+* mailto:jdenise@redhat.com[Jean-François Denise]
+
+=== QE Contacts:
+
+* mailto:eduda@redhat.com[Erich Duda]
+
+=== Affected Projects or Components:
+
+* WildFly Core CLI
+
+
+== Requirements
+
+=== Hard requirements
```

Comment:
Can you please elaborate what type the pattern is? Is it simple string or regular expression?

Reply:
Hmm, as Linux/Git user, I am used to regular expressions when I search text in long outputs (e.g. `git log`). Is there some particular reason why you chose simple text?

Reply:
The reason is to keep the feature surface small, really address the minimal search requirements. If that is a too big limitation, that would make the feature unusable, then this could be revisited.

Reply:
I see. IMO it is not big limitation and it can be added later. Thank you.